DOCUMENTATION.md

```diff
@@ -0,0 +1,116 @@
+Printing system | `--socket=cups` and `--nosocket=cups` | Toggle | Permit access to the application to printing systems (`--socket=cups`); prohibit access to the application to printing systems (`--nosocket=cups`).
+
+#### Device
+
+List of devices available in the sandbox.
+
+Name | Permission(s) | Type | Description
+--- | --- | --- | ---
+GPU acceleration | `--device=dri` and `--nodevice=dri` | Toggle | Permit access to the application to graphics direct rendering located at `/dev/dri` (`--device=dri`); prohibit access to the application to graphics direct rendering (`--nodevice=dri`).
+Virtualization | `--device=kvm` and `--nodevice=kvm` | Toggle | Permit access to the application to virtualization (`--device=kvm`); prohibit access to the application to virtualization (`--nodevice=kvm`).
+Shared memory | `--device=shm` and `--nodevice=shm` | Toggle | Permit access to the application to shared memory (`--device=shm`); prohibit access to the application to shared memory (`--nodevice=shm`).
+All devices | `--device=all` and `--nodevice=all` | Toggle | Permit access to the application to all devices (`--device=all`); prohibit access to the application to all devices (`--nodevice=all`).
+
+#### Allow
+
+List of features available to the application.
+
+Name | Permission(s) | Type | Description
+--- | --- | --- | ---
+Development syscalls | `--allow=devel` and `--disallow=devel` | Toggle | [NEEDS DESCRIPTION]
+Programs from other architectures | `--allow=multiarch` and `--disallow=multiarch` | Toggle | [NEEDS DESCRIPTION]
+Bluetooth | `--allow=bluetooth` and `--disallow=bluetooth` | Toggle | Permit access to use bluetooth (`--allow=bluetooth`); prohibit access to use bluetooth (`--disallow=bluetooth`).
+Controller Area Network bus | `--allow=canbus` and `--disallow=canbus` | Toggle | [NEEDS DESCRIPTION]
+
+#### Filesystem
+
+List of filesystem subsets available to the application.
+
+Name | Permission(s) | Type | Description
+--- | --- | --- | ---
+All filesystem files | `--filesystem=host` and `--nofilesystem=host` | Toggle | Permit read-write access to the application to the whole filesystem (excluding directories or files owned by other groups) (`--filesystem=host`); prohibit read-write acess to the application to the whole filesystem (`--nofilesystem=host`). 
+All system libraries, executables and static data | `--filesystem=host-os` and `--nofilesystem=host-os` | Toggle | [NEEDS DESCRIPTION]
+All system configurations | `--filesystem=host-etc` and `--nofilesystem=host-etc` | Toggle | [NEEDS DESCRIPTION]
+All user files | `--filesystem=home` and `--nofilesystem=home` | Toggle | Permit read-write access to the application to the user directory (`$HOME`) (`filesystem=home`); prohibit read-write access to the application to the user directory (`$HOME`) (`--nofilesystem=home`). 
+Other files | `--filesystem=[DIRECTORY]`, `--filesystem=[DIRECTORY]:ro` and `--nofilesystem=[DIRECTORY]` | Input | [NEEDS DESCRIPTION]
```

Comment:
The description here could focus on examples and what they mean, I think is easier to understand that why, what do you think ?

Reply:
I support your suggestion. I was thinking of the same but I had no idea how to write it. I personally think that we should give examples only to the "Other files" section because it is the only one without a toggle, which is why it needs more explanation.

I'll think of what I can do and will update it in a day or two, since I am busy with school.

Reply:
> I support your suggestion. I was thinking of the same but I had no idea how to write it. I personally think that we should give examples only to the "Other files" section because it is the only one without a toggle, which is why it needs more explanation.
> 

:+1: 

> I'll think of what I can do and will update it in a day or two, since I am busy with school.